pkg/types/clusterconfig/cluster_config_aws.go

```diff
@@ -595,6 +590,13 @@ var AccessValidation = &cr.StructValidation{
 				Validator: RegionValidator,
 			},
 		},
+		{
+			StructField: "Namespace",
+			StringValidation: &cr.StringValidation{
+				Default:       "cortex",
+				AllowedValues: []string{"cortex"},
```

Comment:
Is there a particular reason for using not using the `default` namespace?

Reply:
No particular reason. Since we _manage_ it, we can probably go with `default`. I had it more set to `cortex` for testing purposes.